cfme/tests/services/test_service_catalogs.py

```diff
@@ -190,7 +189,8 @@ def test_no_template_catalog_item(provider_crud, provider_type, provisioning,
     catalog_item = CatalogItem(item_type=catalog_item_type, name=item_name,
                   description="my catalog", display_in=True, catalog=catalog, dialog=dialog)
     catalog_item.create()
-    flash.assert_message_match("'Catalog/Name' is required")
+    flash.assert_message_match(version.pick({version.LOWEST: "Source_id must have valid template",
```

Comment:
@sshveta This seems counter to what you had before? ```Source_id must have valid template``` was the _addition_ but as it's written here, it would be come the ```5.2``` version. Is this right?